intermediate_source/torchvision_tutorial.rst

```diff
@@ -103,13 +106,21 @@ Here is one example of a pair of images and segmentation masks
 So each image has a corresponding
 segmentation mask, where each color correspond to a different instance.
 Let’s write a ``torch.utils.data.Dataset`` class for this dataset.
+In the code below we are wrapping images, bounding boxes and masks into
+``torchvision.datapoints`` structures such that we will be able apply torchvision
+built-in transformations (`new Transforms API <https://pytorch.org/vision/stable/transforms.html>`_)
+that cover the object detection and segmetation tasks.
+More on torchvision datapoints see this `documentation <https://pytorch.org/vision/stable/datapoints.html>`_
```

Comment:
```suggestion
In the code below, we are wrapping images, bounding boxes and masks into
``torchvision.datapoints`` structures so that we will be able to apply torchvision
built-in transformations (`new Transforms API <https://pytorch.org/vision/stable/transforms.html>`_)
that cover the object detection and segmetation tasks.
For more information about torchvision datapoints, see `this documentation <https://pytorch.org/vision/stable/datapoints.html>`_
```